pwa-devdocs/src/peregrine/reference/targets/wrappable-talons/index.md

```diff
@@ -62,7 +60,5 @@ The reference doc content is generated automatically from the source code.
 To update this section, update the doc blocks in the source code
 -->
 
-{% include auto-generated/peregrine/lib/targets/TalonWrapperConfig.md %}
```

Comment:
This file no longer exists since the recent extensibility update. So I need to remove this line so the site will build properly. I could make this change in a separate PR, but it's very minor.